presentation/components/src/test/instance-filter-builder/InstanceFilterConverter.test.ts

```diff
@@ -0,0 +1,477 @@
+
+      it("'>'", async () => {
+        const filter: PresentationInstanceFilterCondition = {
+          field,
+          operator: PropertyFilterRuleOperator.Greater,
+          value,
+        };
+        const { expression } = await convertToInstanceFilterDefinition(filter, testImodel);
+        expect(expression).to.be.eq(`${propertyAccessor} > 1`);
+      });
+
+      it("'>='", async () => {
+        const filter: PresentationInstanceFilterCondition = {
+          field,
+          operator: PropertyFilterRuleOperator.GreaterOrEqual,
+          value,
+        };
+        const { expression } = await convertToInstanceFilterDefinition(filter, testImodel);
+        expect(expression).to.be.eq(`${propertyAccessor} >= 1`);
+      });
+
+      it("'<'", async () => {
+        const filter: PresentationInstanceFilterCondition = {
+          field,
+          operator: PropertyFilterRuleOperator.Less,
+          value,
+        };
+        const { expression } = await convertToInstanceFilterDefinition(filter, testImodel);
+        expect(expression).to.be.eq(`${propertyAccessor} < 1`);
+      });
+
+      it("'<='", async () => {
+        const filter: PresentationInstanceFilterCondition = {
+          field,
+          operator: PropertyFilterRuleOperator.LessOrEqual,
+          value,
+        };
+        const { expression } = await convertToInstanceFilterDefinition(filter, testImodel);
+        expect(expression).to.be.eq(`${propertyAccessor} <= 1`);
+      });
+
+      it("'Like'", async () => {
+        const filter: PresentationInstanceFilterCondition = {
+          field,
+          operator: PropertyFilterRuleOperator.Like,
+          value: { valueFormat: PropertyValueFormat.Primitive, value: `someString` },
+        };
+        const { expression } = await convertToInstanceFilterDefinition(filter, testImodel);
+        expect(expression).to.be.eq(`${propertyAccessor} LIKE "%someString%"`);
+      });
+    });
+
+    it("quoted string value", async () => {
+      const filter: PresentationInstanceFilterCondition = {
+        field,
+        operator: PropertyFilterRuleOperator.IsEqual,
+        value: { ...value, value: `string "with" quotation marks` },
+      };
+      const { expression } = await convertToInstanceFilterDefinition(filter, testImodel);
+      expect(expression).to.be.eq(`${propertyAccessor} = "string ""with"" quotation marks"`);
+    });
+
+    it("instance key value", async () => {
+      const propertyInfo = createTestPropertyInfo({ type: "navigation" });
+      const filter: PresentationInstanceFilterCondition = {
+        field: createTestPropertiesContentField({ properties: [{ property: propertyInfo }] }),
+        operator: PropertyFilterRuleOperator.IsEqual,
+        value: { ...value, value: { className: "TestSchema:TestClass", id: "0x1" } },
+      };
+      const { expression } = await convertToInstanceFilterDefinition(filter, testImodel);
+      expect(expression).to.be.eq(`${propertyAccessor} = 0x1`);
```

Comment:
What would be the value of `propertyAccessor` for navigation properties? I think it needs an additional `.Id` suffix.